From 89fc62b849dc59398ff348d6b958d80d658b8de3 Mon Sep 17 00:00:00 2001
From: Isabel Paredes <isabel.paredes@quantstack.net>
Date: Fri, 25 Apr 2025 14:25:50 +0200
Subject: [PATCH 1/2] Add no-implicit-function-declaration flag

gfortran option for LAPACK to improve thread safety
It is enabled by default in Makefile.system for gfortran
---
 Makefile.rule | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.rule b/Makefile.rule
index c3edd1c..950a1fc 100644
--- a/Makefile.rule
+++ b/Makefile.rule
@@ -256,7 +256,7 @@ NO_AFFINITY = 1
 # Common Optimization Flag;
 # The default -O2 is enough.
 # Flags for POWER8 are defined in Makefile.power. Don't modify COMMON_OPT
-# COMMON_OPT = -O2
+COMMON_OPT = -O2 -Wno-implicit-function-declaration
 
 # gfortran option for LAPACK to improve thread-safety
 # It is enabled by default in Makefile.system for gfortran
